configuration/enabling-desktop-icons/en.md

```diff
@@ -3,12 +3,34 @@ title = "Enabling Desktop Icons"
 +++
 # Enabling Desktop Icons
 
-You can enable desktop icons by opening Raven, clicking the cog wheel for Budgie Settings, then toggling Desktop Icons on under the Background section.
+You can enable or disable desktop icons easily depending on your personal preference.
+
+## Budgie
+
+With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Windows Key + A`.  Next, click the cog icon for Budgie Settings and then toggle "Desktop Icons" on under the Background section.
 
 {{< altimg "raven-desktop-icons.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
 
-You may now add things to your Desktop by:
-1. Opening the file manager
-2. Moving the files you want to the Desktop folder (as shown in the image below).
+You may now add things to your Desktop by opening the file manager and then moving the files you want to the Desktop folder (as shown in the image below).
 
 {{< altimg "nautilus-desktop-folder.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+## GNOME
+
+In GNOME, you can enable desktop items by opening the Applications Overlay using the keyboard shortcut `Windows Key + A` or by clicking the Applications icon in the dock and then search for and open "Tweak Tool".  You can then toggle them on and off by clicking "Icons on Desktop" under the Desktop section.
+
+{{< altimg "gnome-tweak-tool-desktop.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+You may now add things to your Desktop by opening the file manager and then moving the files you want to the Desktop folder (as shown in the image below).
+
+{{< altimg "nautilus-desktop-folder.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+## MATE
+
+MATE has desktop icons enabled by default.  If you wish to disable or re-enable them, open the Brisk Menu, click on "Preferences" and then on "MATE Tweak".  You can then toggle them on and off by clicking "Show Desktop Icons" under the Desktop section.
+
+{{< altimg "mate-tweak-desktop.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+You may now add things to your Desktop by opening the Caja file manager and then moving the files you want to the Desktop folder (as shown in the image below).
```

Comment:
If you have desktop icons enabled you don't need to use the file manager, as the desktop _is the file manager_. You can right click and drag etc